src/commands.c

```diff
@@ -1572,6 +1814,64 @@ int yh_com_put_authentication(yubihsm_context *ctx, Argument *argv,
   return 0;
 }
 
+// NOTE(adma): Store an asymmetric authentication key
```

Comment:
I think we can drop the (adma) here.

Reply:
Fixed

Reply:
See commit 23c464a8f8c92445ad4b85c271abafac5eb6e0ce